src/main/java/helpedfunctions/PropertiesUtils.java

```diff
@@ -0,0 +1,18 @@
+package helpedfunctions;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesUtils {
+
+    public static Properties loadProperties(String propFilePath) throws IOException {
+        Properties properties = new Properties();
+        try (InputStream in = new FileInputStream(propFilePath)) {
+            properties.load(in);
+        }
+
+        return properties;
+    }
```

Comment:
add here fucntion to read chromedriver property 
"chromedriver" can be as key, as constant 
private static String CHROMEDRIVER_KEY="chromedriver"
System.getProperty(CHROMEDRIVER_KEY, properties.getProperty(CHROMEDRIVER_KEY));